demos/text_detection_demo/text_detection_demo.hpp

```diff
@@ -51,8 +51,8 @@ DEFINE_string(m_td, "", text_detection_model_message);
 DEFINE_string(m_tr, "", text_recognition_model_message);
 DEFINE_string(m_tr_ss, "0123456789abcdefghijklmnopqrstuvwxyz", text_recognition_model_symbols_set_message);
 DEFINE_bool(cc, false, text_central_image_crop_message);
-DEFINE_int32(w_td, 1280, image_width_for_text_detection_model_message);
-DEFINE_int32(h_td, 768, image_height_for_text_detection_model_message);
+DEFINE_int32(w_td, 0, image_width_for_text_detection_model_message);
+DEFINE_int32(h_td, 0, image_height_for_text_detection_model_message);
```

Comment:
This changes could affect existing behavior. Could you explain it necessity of these changes?

Reply:
I doubt, that this modification will affect existing behavior because of this [line](https://github.com/GalyaZalesskaya/open_model_zoo/blob/da991e8cb0b82c714adad571291725c779dead0a/demos/text_detection_demo/src/cnn.cpp#L29)  in the text detection model initialization.
It means that size of text detection model is calculated directly from model information ([line](https://github.com/GalyaZalesskaya/open_model_zoo/blob/da991e8cb0b82c714adad571291725c779dead0a/demos/text_detection_demo/src/cnn.cpp#L27)) if some special resolution is not set with parameters (new_input_resolution!=cv::Size()). 

It is needed not to pass internal architecture information (that user doesn't need to know) through parameters and make demo more generalized for models with different input resolution.
I checked that behavior for text-detection-0004 did not change, because (768, 1280) resolution is already define with model, so you don't need to keep default shape in this file.
Please say if my check is not enough.